libpod/events/logfile.go

```diff
@@ -1,15 +1,23 @@
+//go:build linux
+// +build linux
+
 package events
 
 import (
+	"bufio"
 	"context"
 	"fmt"
+	"io"
+	"io/ioutil"
 	"os"
 	"time"
 
 	"github.com/containers/podman/v4/pkg/util"
 	"github.com/containers/storage/pkg/lockfile"
+       "github.com/nxadm/tail"
```

Comment:
Just noticed now; the indentation is off here.

Curious why this passes `make validate`.